src/popups/AddProject/AddProject.scss

```diff
@@ -0,0 +1,47 @@
+.heading{
+    border-bottom:0 none !important;
```

Comment:
When an important rule is used on a style declaration, this declaration overrides any other declarations. Although technically !important has nothing to do with specificity, it interacts directly with it. Using !important, however, is bad practice and should be avoided because it makes debugging more difficult by breaking the natural cascading in your stylesheets. When two conflicting declarations with the !important rule are applied to the same element, the declaration with a greater specificity will be applied.

>

Reply:
Actually i used !important to remove the default header border that comes as default with <Modal.header>
![Screenshot from 2020-02-22 21-00-41](https://user-images.githubusercontent.com/46385227/75095006-814df480-55b6-11ea-8cf1-6f67ade55962.png)

Hope you can notice the header border below "About the Project" .If you feel it looks good with header border then i will  remove  !important .

Reply:
I think there should be a way to do this without imp 👍 .
1 Looks better :)